This program is not very useful as a downstream, it seems to be only intended
for internal QA. It also has a lot of weird things like hardcoding the prefixed
cpp to use and putting a text file in /usr/bin.

piolib is apparently still heavily in development so it doesn't make sense to
install for now

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,10 +5,8 @@
 # List of subsidiary CMakeLists
 add_subdirectory(dtmerge)
 add_subdirectory(otpset)
-add_subdirectory(overlaycheck)
 add_subdirectory(ovmerge)
 add_subdirectory(pinctrl)
-add_subdirectory(piolib)
 add_subdirectory(raspinfo)
 add_subdirectory(vcgencmd)
 add_subdirectory(vclog)
